app/models/performance_statistics.rb

```diff
@@ -129,6 +129,14 @@ def ended_without_success_application_form_status_total_counts
     application_form_status_total_counts(only: %i[ended_without_success])
   end
 
+  def ended_without_success_application_withdrawn_at_candidate_request_count
+    @withdrawn_at_candidates_request_count ||= GetApplicationChoicesWithdrawnAtCandidatesRequest.call.count
+  end
+
+  def ended_without_success_application_withdrawn_by_candidate_count
+    application_form_status_total_counts(only: %i[withdrawn_at_candidates_request])
+  end
+
```

Comment:
Are these used?

Reply:
@benswannack  Oops no - this was committed in error, part of an earlier attempt, removed here https://github.com/DFE-Digital/apply-for-teacher-training/pull/5201/commits/0e0a185a427759b1e84ca85cb2200f2b7ca4c6fb